intel-retail/automated-vending
# Fix: Update to actions template to match the current repo

chore(deps): Update EdgeX Dependencies to 3.1.0

## PR Checklist



## What are you changing?


Update github actions to match the current repo

## Issue this PR will close

None

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

1. Start a new PR for a full run
2. Run each action manually
3. Check main readme badges

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-vending )
